Dockerfile

```diff
@@ -6,6 +6,7 @@ FROM centos:centos7.9.2009
 #             https://mybinder.readthedocs.io/en/latest/tutorials/dockerfile.html
 ARG NB_USER=notebook-user
 ARG NB_UID=1000
+ARG PIP_VERSION=22.3.1
```

Comment:
I'd actually remove the default here since this is intended to always be explicitly [passed with scripts/docker-build.sh](https://github.com/Unstructured-IO/unstructured-inference/blob/main/scripts/docker-build.sh) .

Reply:
sounds good